src/modules/utils/modals/assetInfo/assetInfo.less

```diff
@@ -19,7 +19,11 @@ html body md-dialog.asset-info-modal {
       flex-direction: row;
       justify-content: space-between;
       border-bottom: 1px solid @color-disabled-100;
+      padding: 5px 0;
 
+      &:last-child {
+        border-bottom: 0 none;
+      }
```

Comment:
We try to keep the code style where there is a blank line between two LESS blocks.
Add a line, please, and update.